src/main/java/Onandon/module/Task.java

```diff
@@ -0,0 +1,32 @@
+package Onandon.module;
+
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "X" : " "); // mark done task with X
```

Comment:
Try to remove redundant comments. More on this will be covered in [Week 7 - Code Quality - Comments](https://nus-cs2113-ay2223s2.github.io/website/schedule/week7/topics.html#w7-5-code-quality-code-comments)